### PR TITLE
add link to file github issue directly from demo

### PR DESCRIPTION
**Pull request by dlyongemallo**

### CODE REVIEW

**Comment by dlyongemallo** [comment]
This is on top of PR #934.

File: java/demo/src/com/google/phonenumbers/PhoneNumberParserServlet.java [hunk 1]
```diff
@@ -164,6 +157,39 @@ private void appendLine(String title, String data, StringBuilder output) {
...
+        .append(phoneNumber).append("\n");
+    issueTemplate.append(
+        "The phone number range(s) to which the issue applies (\"+1 555 555-XXXX\"): \n");
+    issueTemplate.append(
+        "The type of the number(s) (\"fixed-line\", \"mobile\", \"short code\", etc.): \n");
+    issueTemplate.append(
+        "The cost, if applicable (\"toll-free\", \"premium rate\", \"shared cost\"): \n");
+    issueTemplate.append(
+        "Supporting evidence (for example, national numbering plan, announcement from mobile "
+        + "carrier, news article): [IMPORTANT - anything posted here is made public. "
```
Comment (dlyongemallo) [inline java/demo/src/com/google/phonenumbers/PhoneNumberParserServlet.java]: As discussed, I added another warning here. @roes

**Comment by roes** [comment]
This looks fine to me, would like to see a demo before merging though.

**Comment by roes** [comment]
Looks good to me, go ahead and merge.

**Comment by keghani** [comment]
Released in 7.2.3

**Pull request merged**